data/unihannover/unihannover.csv

```diff
@@ -68,3 +68,36 @@ Hannover U,2015,148.75,10.5194/ars-13-175-2015,FALSE,,,,
 Hannover U,2015,297.5,10.5194/ars-13-9-2015,FALSE,,,,
 Hannover U,2015,1578.8,10.3389/fmicb.2015.01224,FALSE,,,,
 Hannover U,2015,1426.04,10.1063/1.4923379,FALSE,,,,
+Hannover U,2016,1856.4,10.1088/1367-2630/18/1/013050,FALSE,NA,NA,NA,NA
+Hannover U,2016,1713.6,10.5194/hess-2016-17,FALSE,NA,NA,NA,NA
+Hannover U,2016,185.64,10.5194/ars-14-115-2016,FALSE,NA,NA,NA,NA
+Hannover U,2016,247.52,10.5194/ars-14-17-2016,FALSE,NA,NA,NA,NA
+Hannover U,2016,185.64,10.5194/ars-14-91-2016,FALSE,NA,NA,NA,NA
+Hannover U,2016,123.76,10.5194/ars-14-51-2016,FALSE,NA,NA,NA,NA
+Hannover U,2016,123.76,10.5194/ars-14-47-2016,FALSE,NA,NA,NA,NA
+Hannover U,2016,265.54,10.1016/j.dib.2016.01.067,FALSE,NA,NA,NA,NA
+Hannover U,2016,1253.47,10.3390/su8020178,FALSE,NA,NA,NA,NA
+Hannover U,2016,1559.28,10.3389/fmicb.2016.00315,FALSE,NA,NA,NA,NA
+Hannover U,2016,1418.94,10.1063/1.4944839,FALSE,NA,NA,NA,NA
+Hannover U,2016,1211,10.1127/metz/2016/0797,FALSE,NA,NA,NA,NA
+Hannover U,2016,1118.6,10.1186/s13561-016-0094-6,FALSE,NA,NA,NA,NA
+Hannover U,2016,1594.18,10.1155/2016/7372306,FALSE,NA,NA,NA,NA
+Hannover U,2016,1492.12,10.1097/MD.0000000000003961,FALSE,NA,NA,NA,NA
+Hannover U,2016,1456.56,10.1002/ece3.2258,FALSE,NA,NA,NA,NA
+Hannover U,2016,2025.01,10.3389/fpls.2016.00875,FALSE,NA,NA,NA,NA
+Hannover U,2016,1628.31,10.2196/ijmr.5822,FALSE,NA,NA,NA,NA
+Hannover U,2016,1856.4,10.1088/1367-2630/18/7/073014,FALSE,NA,NA,NA,NA
+Hannover U,2016,1983.37,10.3389/fpls.2016.00995,FALSE,NA,NA,NA,NA
+Hannover U,2016,1392.3,10.1088/1367-2630/18/9/093012,FALSE,NA,NA,NA,NA
+Hannover U,2016,1726.69,10.1186/s12971-016-0097-4,FALSE,NA,NA,NA,NA
+Hannover U,2016,1588.1,10.1371/journal.pone.0163679,FALSE,NA,NA,NA,NA
+Hannover U,2016,565.25,10.1186/s40552-016-0037-0,FALSE,NA,NA,NA,NA
+Hannover U,2016,1820.7,10.1002/ece3.2484,FALSE,NA,NA,NA,NA
+Hannover U,2016,1386.35,10.1038/srep35207,FALSE,NA,NA,NA,NA
+Hannover U,2016,1118.6,10.1186/s13561-016-0130-6,FALSE,NA,NA,NA,NA
+Hannover U,2016,1610,10.1016/j.pacs.2016.10.002,FALSE,NA,NA,NA,NA
+Hannover U,2016,995.77,10.3390/ijgi5110208,FALSE,NA,NA,NA,NA
+Hannover U,2016,1182,10.1186/s12860-016-0116-6,FALSE,NA,NA,NA,NA
+Hannover U,2016,1322.87,10.1109/TPEL.2016.2537372,TRUE,NA,NA,NA,NA
+Hannover U,2016,1322.87,10.1109/TEMC.2016.2554463,TRUE,NA,NA,NA,NA
+Hannover U,2016,1020.50,10.1109/JPHOT.2016.2570001,TRUE,NA,NA,NA,NA
```

Comment:
@tullney Are you sure this is a hybrid publication? "IEEE Photonics Journal" is reported as fully OA by several other institutions and it's also listed in the DOAJ.